lib/logstash/inputs/http.rb

```diff
@@ -75,11 +83,20 @@ def register
   end # def register
 
   def run(queue)
```

Comment:
Also as we move to an API based backend we will probably use some sort of rack compatible controllers like sinatra.